mne/viz/backends/_pyvista.py

```diff
@@ -658,6 +659,10 @@ def _add_mesh(plotter, *args, **kwargs):
         smooth_shading = kwargs.pop('smooth_shading')
     else:
         smooth_shading = True
+    # disable rendering pass for add_mesh, render()
+    # is called in show()
+    if 'render' in _get_args(plotter.add_mesh):
```

Comment:
For future compat in case at some point we want to pass `render=True` for some reason
```suggestion
    if 'render' not in kwargs and 'render' in _get_args(plotter.add_mesh):
```